bower.json

```diff
@@ -7,6 +7,7 @@
     "html5shiv": "~3.7.3",
     "jquery": "~2.1.4",
     "normalize-css": "~3.0.3",
-    "respond": "~1.4.2"
+    "respond": "~1.4.2",
+    "bootstrap-sass": "bootstrap-sass-official#~3.3.5"
```

Comment:
No worries. It isn't doing any harm for now if it's going to be part of something else in the near future.